js/ui/drop_down_editor/ui.drop_down_list.js

```diff
@@ -761,7 +763,8 @@ const DropDownList = DropDownEditor.inherit({
         this.option('opened', shouldOpenPopup);
 
         if(shouldOpenPopup) {
-            this._dimensionChanged();
+            this._updatePopupWidth();
+            this._popup && this._updatePopupDimensions();
```

Comment:
Let's move `this._popup` condition  inside of `_updatePopupDimensions` to reduce copypaste

Reply:
https://github.com/DevExpress/DevExtreme/pull/23483/commits/709cfa030c245fe825d90854ffcbc9655bae3ca2